jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -228,7 +228,13 @@ def _jwst2data(file_obj, ext, data_label):
 
         imdata = getattr(dm, ext)
         component = Component.autotyped(imdata, units=bunit)
-        data.add_component(component=component, label=comp_label)
+
+        # Might have bad GWCS. If so, we exclude it.
+        try:
+            data.add_component(component=component, label=comp_label)
+        except Exception:  # pragma: no cover
```

Comment:
I cannot think of a simple way to test this without a lot of mocking or requesting some test data that is not too big and too secretive.